Codeinwp/neve
# Automatically set full width to checkout page [#718]

### Summary
Sets the checkout page to full width by default.


### Will affect visual aspect of the product

NOt sure

### Test instructions

- Install WooCommerce
- Go through setup wizard which creates the shop pages 
- Checkout page should be full width by default
- Metabox settings should still work as expected


Closes #718.
